lms/static/scripts/frontend_apps/components/FileList.js

```diff
@@ -5,8 +5,27 @@ import propTypes from 'prop-types';
 import Spinner from './Spinner';
 import Table from './Table';
 
+/**
+ * @typedef File
+ * @prop {string} display_name - The filename
+ * @prop {string} updated_at - An ISO date or date + time string
```

Comment:
It will operate successfully on any ISO date[time] string?

Reply:
Technically anything that the `Date` constructor can parse, but the API should always provide an ISO date-time string.